app/styles/_pipeline.less

```diff
@@ -0,0 +1,167 @@
+@build-pipeline-border-color: #d1d1d1;
+
+// Statuses
+@build-pipeline-new-color: #bee1f4;
+@build-pipeline-pending-color: #d1d1d1;
+@build-pipeline-in-progress-color: #0088ce;
+@build-pipeline-success-color: #3f9c35;
+@build-pipeline-failed-color: #cc0000;
+@build-pipeline-aborted-color: #ededed;
+
+.build-pipeline {
+  margin-bottom: 15px;
+  @media (min-width: 480px) {
+    margin-bottom: 0;
+  }
+  .build-summary, .stage {
+    .text-center();
+    min-width: 150px;
+  }
+  .build-summary, .stages-block {
+    border: 1px solid @build-pipeline-border-color;
+  }
+  .build-summary {
+    @media (min-width: 480px) {
+      border-right-width: 0;
+      flex-grow: 0;
+      flex-shrink: 0;
+      flex-basis: 200px;
+    }
+    min-height: 100px;
+    padding: 10px;
+    .status-icon {
+      margin-bottom: 3px;
+      .inverse {
+        font-size: 8px;
+        margin-bottom: 3px;
+      }
+      &.Complete .inverse {
```

Comment:
upper case class names feel wrong, is this because you are using a class that matches the status we get back?  could we do what we use to do with pods and do .statusComplete .statusFailed,  or just lowercase the status?